docs/guides/genomics-platform/analyzing-data/sequencerr.md

```diff
@@ -80,11 +69,60 @@ You can navigate to the **SequencErr** application [here](https://platform.stjud
     Set this to prevent large count files from being generated
     Default: true
 
-**Provide Input Parameters and Run the Analysis**
-![](../../../files/guides/tools/sequencerr/SequencErr_steps_7a-10.png)
+## Running the Analysis
+
+**SequencErr** application can be accessed from [https://platform.stjude.cloud/workflows/sequencerr](https://platform.stjude.cloud/workflows/sequencerr)
+
+!!! note
+    This tool is intended free-of-charge for non-profit usages. 
+    Please contact [Dr. Xiaotu Ma](mailto:Xiaotu.Ma@stjude.org) for for-profit usages and modifications
+
+Please refer to the stepwise **instructions PDF [here](../../../files/guides/tools/sequencerr/SequencErr_Instructions.pdf)** or following steps to learn how to launch the workflow, hook up input files, adjust parameters, run analysis and inspect output files.
+
+**Log in and Launch the (SequencErr)[https://platform.stjude.cloud/workflows/sequencerr]**
+
+![](../../../files/guides/tools/sequencerr/Sequencerr_log_in_1.png)
+
+![](../../../files/guides/tools/sequencerr/Sequencerr_log_in_2.png)
+
+![](../../../files/guides/tools/sequencerr/Sequencerr_log_in_3.png)
+
+![](../../../files/guides/tools/sequencerr/Sequencerr_launch_4.png)
+
+
+**Choose Input BAM and Index Files**
+
+Users can upload and download data files with the help of **Data Transfer App** as described [here](../../../guides/genomics-platform/managing-data/data-transfer-app.md) 
+
+![](../../../files/guides/tools/sequencerr/Sequencerr_BAM_5.png)
+
+![](../../../files/guides/tools/sequencerr/Sequencerr_BAM_6.png)
+
+![](../../../files/guides/tools/sequencerr/Sequencerr_BAM_7.png)
+
+![](../../../files/guides/tools/sequencerr/Sequencerr_BAM_8.png)
+
+Follow the similar steps to choose and select the corresponding BAM index file from SequencErr Demo project
```

Comment:
Only the reviewers will have access to the SequencErr Demo project. I recommend taking this reference out, since only the reviewers will understand it.